faiadashu_online/pubspec.yaml

```diff
@@ -18,11 +18,11 @@ dependencies:
   faiadashu:
     path: ..
 
-  fhir: ^0.8.0
+  fhir: ^0.11.0
 
-  fhir_at_rest: ^0.8.3
+  fhir_at_rest: ^0.11.0
 
-  fhir_auth: ^0.8.0
+  fhir_auth: ^0.11.0-dev.1
```

Comment:
Is it intended to use `dev` version?

Reply:
Thanks for the explanation.

I think we can keep it as is since `faiadashu_online` is not used by anyone.
We can resolve this later if someone is raising the issue.